### PR TITLE
When canceling an app update, rewrite the last active release on top

### PR DESCRIPTION
**Pull request by heronrs**
When canceling an app update, rewrite the last active release on top of the canceled one.

### CODE REVIEW

**Review by Twsouza — COMMENTED** [review]
@heronrs should we add a description to the canceled release, something like `Release canceled - ...`? Or some way that we can identify that the build was canceled?

File: pkg/cli/apps.go [hunk 1]
```diff
@@ -109,13 +109,33 @@ func Apps(rack sdk.Interface, c *stdcli.Context) error {
...
+	if err != nil {
 		return err
 	}
+	for _, r := range rs {
+		if app.Release == r.Id {
+			_, err := rack.ReleaseCreate(app.Name, structs.ReleaseCreateOptions{Build: &r.Build, Description: &r.Description, Env: &r.Env})
+			if err != nil {
+				return err
+			}
+		}
```
Comment (Twsouza) [inline pkg/cli/apps.go:137]: maybe add a break statement when finding the release.

**Review by Twsouza — CHANGES_REQUESTED** [review]

**Comment by heronrs** [comment]
> @heronrs should we add a description to the canceled release, something like `Release canceled - ...`? Or some way that we can identify that the build was canceled?

@Twsouza I think this one would be nice to have but since it involves a bit more work - we'd need to create a rack endpoint specifically for updating a release - I think we can do it later in a different task.

**Review by Twsouza — APPROVED** [review]
:fire:

**Pull request merged**